### PR TITLE
Update backwards-compatibility.md

### PR DESCRIPTION
**Pull request by rexstjohn-anza**
#### Summary of Changes

Fixes #

### CODE REVIEW

**Comment by rexstjohn-anza** [comment]
I have questions on this one:

### Rust Crates
- [`solana-sdk`](https://docs.rs/solana-sdk/) - Rust SDK for creating transactions and parsing account state
- [`solana-program`](https://docs.rs/solana-program/) - Rust SDK for writing programs
- [`solana-client`](https://docs.rs/solana-client/) - Rust client for connecting to RPC API
- [`solana-cli-config`](https://docs.rs/solana-cli-config/) - Rust client for managing Solana CLI config files
- [`agave-geyser-plugin-interface`](https://docs.rs/agave-geyser-plugin-interface/) - Rust interface for developing Solana Geyser plugins.

Are all these URLS going to remain same for agave 

#### Public API Nodes
Anza provides publicly available RPC API nodes for all developers to use. The Anza team
will make their best effort to communicate any changes to the host, port, rate-limiting behavior,
availability, etc. However, we recommend that developers rely on their own validator nodes to
discourage dependence upon Solana operated nodes.

Accurate?

Are we changing these all to Agave?

Solana software releases include APIs, SDKs, and CLI tooling (with a few [exceptions](https://docs.solanalabs.com/backwards-compatibility#exceptions)).
Solana software releases follow semantic versioning, more details below.
Software for a MINOR version release will be compatible across all software on the same MAJOR version.

**Comment by rexstjohn-anza** [comment]
Will this be "Agave" or Solana?

### Release Cadence

The Solana RPC API, Rust SDK, CLI tooling, and SBF Program SDK are all updated and shipped
along with each Solana software release and should always be compatible between `PATCH`
updates of a particular `MINOR` version release.

**Comment by softalchemy** [comment]
> Anza provides publicly available RPC API nodes for all developers to use. The Anza team
> will make their best effort to communicate any changes to the host, port, rate-limiting behavior,
> availability, etc. However, we recommend that developers rely on their own validator nodes to
> discourage dependence upon Solana operated nodes.

Not accurate anymore, foundation pays Triton for the public endpoint, we don't maintain it since like 3 years ago

**Comment by softalchemy** [comment]
Did @willhickey share the crates migration list? You can use that for reference to know what's changed

**Comment by willhickey** [comment]
> ### Rust Crates
> * [`solana-sdk`](https://docs.rs/solana-sdk/) - Rust SDK for creating transactions and parsing account state
> * [`solana-program`](https://docs.rs/solana-program/) - Rust SDK for writing programs
> * [`solana-client`](https://docs.rs/solana-client/) - Rust client for connecting to RPC API
> * [`solana-cli-config`](https://docs.rs/solana-cli-config/) - Rust client for managing Solana CLI config files
> * [`agave-geyser-plugin-interface`](https://docs.rs/agave-geyser-plugin-interface/) - Rust interface for developing Solana Geyser plugins.
> 
> Are all these URLS going to remain same for agave

Yeah those names are URLs are correct and should stay as they are.

> Are we changing these all to Agave?
> 
> Solana software releases include APIs, SDKs, and CLI tooling (with a few [exceptions](https://docs.solanalabs.com/backwards-compatibility#exceptions)). Solana software releases follow semantic versioning, more details below. Software for a MINOR version release will be compatible across all software on the same MAJOR version.

APIs, SDKs, and CLI names will stay the same. We [renamed 6 crates](https://github.com/anza-xyz/agave/wiki/Agave-Transition#crates-renamed) but the rest stay the same.

File: docs/src/backwards-compatibility.md [hunk 1]
```diff
@@ -3,15 +3,18 @@ title: Backward Compatibility Policy
...
+and so this document attempts to clarify and codify the process for new releases. Furthermore,
+there will be a growing number of validator clients maintained seperately by distinct teams. 
+Coordinating across these teams to ensure the reliability of the network will require ongoing
+communication. 
 
 ### Expectations
 
-- Solana software releases include APIs, SDKs, and CLI tooling (with a few [exceptions](#exceptions)).
-- Solana software releases follow semantic versioning, more details below.
+- Agave software releases include APIs, SDKs, and CLI tooling (with a few [exceptions](#exceptions)).
```
Comment (willhickey) [inline docs/src/backwards-compatibility.md:16]: We will probably decouple these soon, but for now this is still accurate.

There's nothing in the exceptions section so we can get rid of that.

File: docs/src/backwards-compatibility.md [hunk 2]
```diff
@@ -128,7 +131,7 @@ The release process is as follows:
 
 #### Public API Nodes
 
-Solana provides publicly available RPC API nodes for all developers to use. The Solana team
+Anza provides publicly available RPC API nodes for all developers to use. The Anza team
```
Comment (willhickey) [inline docs/src/backwards-compatibility.md]: Based on Joe's comment I think we can remove this whole section.

**Review by rexstjohn-anza — COMMENTED** [review]
Removing bit about provided RPC nodes

File: docs/src/backwards-compatibility.md [hunk 3]
```diff
@@ -3,15 +3,18 @@ title: Backward Compatibility Policy
...
 As the Solana developer ecosystem grows, so does the need for clear expectations around
-breaking API and behavior changes affecting applications and tooling built for Solana.
+breaking API and behavior changes affecting applications and tooling built for Solana by Anza.
 In a perfect world, Solana development could continue at a very fast pace without ever
 causing issues for existing developers. However, some compromises will need to be made
-and so this document attempts to clarify and codify the process for new releases.
+and so this document attempts to clarify and codify the process for new releases. Furthermore,
+there will be a growing number of validator clients maintained seperately by distinct teams. 
+Coordinating across these teams to ensure the reliability of the network will require ongoing
+communication.
```
Comment (willhickey) [inline docs/src/backwards-compatibility.md]:
```suggestion
there will be a growing number of validator clients maintained seperately by distinct teams.
Coordinating across these teams to ensure the reliability of the network will require ongoing
communication.
```

**Review by willhickey — APPROVED** [review]

**Pull request merged**